loops-arrays-objects.js

```diff
@@ -49,9 +67,30 @@ const scores = [
 //  ];
 // ==========================================
 
-
-
-
+console.log("Opdracht 1c:")
+console.log("-------------------------------")
+for (let i = 0; i < scores.length; i++) {
+    switch (true) {
```

Comment:
Bij een true/false vergelijking is het vaak duidelijker om een if/else statement te gebruiken in plaats van een switch, op deze manier werkt het zeker ook goed alleen is het wat minder leesbaar.